zerver/lib/actions.py

```diff
@@ -333,6 +368,7 @@ def do_set_realm_name(realm, name):
     return {}
 
 def do_set_realm_restricted_to_domain(realm, restricted):
+    # type: (Realm, bool) -> Dict[None, None]
```

Comment:
Instead of doing this, I have decided to change code in zerver/views/streams.py which calls those functions, otherwise I get errors from mypy.